test/demo/model.test.js

```diff
@@ -1,48 +1,45 @@
-import Model from 'demo/model';
```

Comment:
I'm 80% sure that the Ada style guide prefers using `'` instead of `"`.  There's no reason to change these without reason, as it makes the diff longer.